neard/src/config.rs

```diff
@@ -694,6 +697,19 @@ pub fn init_configs(
             config.telemetry.endpoints.push(MAINNET_TELEMETRY_URL.to_string());
             config.write_to_file(&dir.join(CONFIG_FILENAME));
 
+            if let Some(url) = download_genesis_url {
+                download_genesis(
+                    &url.to_string(),
+                    &genesis.expect("Must specify genesis file path to download").to_string(),
+                );
+            } else if download {
+                let url = get_genesis_url(&chain_id);
+                download_genesis(
+                    &url,
+                    &genesis.expect("Must specify genesis file path to download").to_string(),
+                );
+            }
+
             let genesis: Genesis = serde_json::from_str(
```

Comment:
If you are changing to hosting mainnet genesis and config this part needs to be changed as well. Then we can potentially just merge this branch into the branch with `"testnet" | "betanet" | "devnet"`

Reply:
I think mainnet genesis should also be hosted, but we can do that later.